semcore/date-picker/src/components/PickerAbstract.jsx

```diff
@@ -196,8 +197,14 @@ class PickerAbstract extends Component {
     };
   }
 
+  onVisibleChange = (visible) => {
+    if (visible && this.asProps.value) {
+      this.handlers.displayedPeriod(this.asProps.value);
```

Comment:
It actually should be done inside of `uncontrolledProps()` of date picker abstract classes

Reply:
moved to uncontrolledProps